crates/holochain_zome_types/src/entry.rs

```diff
@@ -83,7 +83,13 @@ impl Default for GetOptions {
     }
 }
 
-#[derive(PartialEq, Debug, Clone, Copy, Serialize, Deserialize)]
+impl From<GetStrategy> for GetOptions {
+    fn from(strategy: GetStrategy) -> Self {
+        Self { strategy }
+    }
+}
+
+#[derive(PartialEq, Default, Debug, Clone, Copy, Serialize, Deserialize)]
```

Comment:
Code style preference that we're gradually working on making more consistent, please can the derive go on the type and the doc comment goes on top of all of it?